tests/module_test_cases/BioCro_daylength_calculator.csv

```diff
@@ -0,0 +1,3 @@
+input,input,input,output,"description"
+lat,solar_dec,sunrise_sunset_elevation,day_length,NA
+40,1,0.8333,12.2569962155787,"automatically-generated test case"
```

Comment:
I don't think this is the automatically generated test case, which uses 1 for all inputs.